src/Client.php

```diff
@@ -5,9 +5,9 @@
 namespace Laminas\Json\Server;
 
 use Laminas\Http\Client as HttpClient;
-use Laminas\Server\Client as ServerClient;
+use Laminas\Server\ClientInterface;
 
-class Client implements ServerClient
+class Client implements ClientInterface
```

Comment:
IMO not something we should move :thinking:

Reply:
This is backwards compatible. New interface extends deprecated interface.